src/Providers/Stripe/Service.php

```diff
@@ -77,4 +100,74 @@ public function update(string $uid): Payment
 
         return new Payment($response->id, $response->currency, $response->amount, $response->created);
     }
+
+    /**
+     * Creates Stripe customer.
+     *
+     * @param string $email
+     * @param string $paymentMethod
+     * @param string|null $description
+     *
+     * @return Customer
+     */
+    public function createCustomer(string $email, string $paymentMethod, ?string $description = null): Customer
+    {
+        $customer = StripeCustomer::create([
+            'description' => $description,
+            'email'       => $email,
+            'payment_method' => $paymentMethod,
```

Comment:
is this a necessary argument? and is it a token the same as it is for a payment? if so I would call it `token`.